templates/_layouts/docs.html

```diff
@@ -29,14 +29,24 @@
                 </a>
               </div>
 
-              {% macro side_nav_item(url, title, label, type) -%}
+              {% macro side_nav_item(url, title) -%}
                   <li class="p-side-navigation__item">
                     <a class="p-side-navigation__link" href="{{ url }}" {% if (slug and slug in url) or (url == path) %}aria-current="page"{% endif %}>
                       {{ title }}
-                      {% if label %}
+                      {% if url in updatedFeatures %}
                         <span class="p-side-navigation__status">
-                          <span class="p-status-label--{{ type|lower }}">
-                            {{ label|capitalize }}
+                          {% if updatedFeatures[url]=="New" %}
```

Comment:
Here we're checking the url when we should be checking the status, no?

Reply:
No. There is a dictionary that maps URL to status. This is searching for that URL in the dictionary

Reply:
> > It's already looking good! The side navigation matching is a bit off but the what's new page looks good!
> 
> Here's what I mean
> 
> ![image](https://user-images.githubusercontent.com/11927929/222414654-cf23e225-fba4-40be-82f3-0ebf59b5a5f1.png)
> 
> ![image](https://user-images.githubusercontent.com/11927929/222414510-02af0321-dbf2-4d75-8d86-aa200dbbe679.png)
> 
> BTW on that last one @bartaz which tag do you reckon we should show?

We had a discussion about that on today's standup. We will take the most important tag only. And we ensure that by making sure that the most important one is the first one in the YAML file. And I added checking for URLs without the hash for the labels

Reply:
Ah fair enough, the naming just confused me a bit I read `updatedFeatures[url]` as "Access the `url` parameter of `updatedFeatures`" but I see what you're doing!